.agents/skills/code-review/SKILL.md

```
@@ -141,26 +134,26 @@ Prefer the remedy that removes moving pieces over one that spreads the same comp
 
 ## Change sizing
 
-Small, focused changes are easier to review, faster to merge and safer to deploy. Targets:
+Small, focused changes are easier to review, faster to merge, and safer to deploy. Targets:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)